packages/markdown-it-htmyst/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "markdown-it-htmyst",
+  "version": "0.1.3",
+  "sideEffects": false,
+  "license": "MIT",
+  "description": "markdown-it HTML hinter for MyST roles and directives",
+  "author": "Tavin Cole <7685034+tavin@users.noreply.github.com>",
+  "homepage": "https://github.com/executablebooks/mystjs/tree/main/packages/markdown-it-htmyst",
+  "main": "./dist/cjs/index.js",
```

Comment:
In #417 we made quite a few changes for working as ESM packages only (and switching to vitest). I am happy to help out with upgrading here as well when you need it.